src/gui/guiFormSpecMenu.cpp

```diff
@@ -2932,9 +3007,9 @@ void GUIFormSpecMenu::acceptInput(FormspecQuitMode quitmode=quit_mode_no)
 					// no dynamic cast possible due to some distributions shipped
 					// without rtti support in irrlicht
 					IGUIElement * element = getElementFromId(s.fid);
-					gui::IGUIScrollBar *e = NULL;
+					guiScrollBar *e = NULL;
 					if ((element) && (element->getType() == gui::EGUIET_SCROLL_BAR)) {
```

Comment:
`guiScrollBar` is not of type `gui::EGUIET_SCROLL_BAR`.
It is `EGUIET_ELEMENT`:
https://github.com/minetest/minetest/blob/429a98964859b83016f2eb47a47a08ab8dc3c57e/src/gui/guiScrollBar.cpp#L19